ibexa/commerce/4.0.x-dev/manifest.json

```diff
@@ -127,36 +127,38 @@
     "HTTPCACHE_DEFAULT_TTL": "86400",
     "HTTPCACHE_PURGE_SERVER": "http://localhost:80",
     "HTTPCACHE_VARNISH_INVALIDATE_TOKEN": "",
+    "#5": "Uncomment following line if you want to enable translation aware behaviour for HTTP Cache.",
+    "#6": "HTTPCACHE_TRANSLATION_AWARE_ENABLED=true",
     "CACHE_POOL": "cache.tagaware.filesystem",
-    "#5": "When using redis or memcached adapters, you can configure it using DSN:",
-    "#6": "- https://symfony.com/doc/4.4/components/cache/adapters/redis_adapter.html#configure-the-connection",
-    "#7": "- https://symfony.com/doc/4.4/components/cache/adapters/memcached_adapter.html#configure-the-connection",
+    "#7": "When using redis or memcached adapters, you can configure it using DSN:",
+    "#8": "- https://symfony.com/doc/4.4/components/cache/adapters/redis_adapter.html#configure-the-connection",
+    "#9": "- https://symfony.com/doc/4.4/components/cache/adapters/memcached_adapter.html#configure-the-connection",
     "CACHE_DSN": "localhost",
     "CACHE_NAMESPACE": "ezp",
     "DATABASE_CHARSET": "utf8mb4",
     "DATABASE_COLLATION": "utf8mb4_unicode_520_ci",
-    "#8": "Needed by Doctrine Bundle / ORM to avoid it opening connection during situations where there is no service yet.",
-    "#9": "See: https://symfony.com/doc/current/reference/configuration/doctrine.html#doctrine-dbal-configuration",
+    "#10": "Needed by Doctrine Bundle / ORM to avoid it opening connection during situations where there is no service yet.",
```

Comment:
```suggestion
    "#10": "Needed by Doctrine Bundle / ORM to prevent it from opening a connection during situations where there is no service yet.",
```